polars/polars-arrow/Cargo.toml

```diff
@@ -10,7 +10,7 @@ description = "Arrow interfaces for Polars DataFrame library"
 
 [dependencies]
 arrow.workspace = true
-chrono = { version = "0.4", default-features = false, features = ["std"], optional = true }
+chrono = { version = "0.4", default-features = false, features = ["std"] }
```

Comment:
Could we make this optional. In that case `PolarsTimeZone` will be:

```rust
#[not(cfg(feature="timezones"))]
pub trait PolarsTimeZone: std::fmt::Debug + std::fmt::Display {}

#[cfg(feature="timezones")]
pub trait PolarsTimeZone: chrono::TimeZone + std::fmt::Debug + std::fmt::Display {}
```

Reply:
thanks! I've tried:
```Rust
#[cfg(feature = "timezones")]
use chrono_tz::Tz;

#[cfg(not(feature = "timezones"))]
pub trait PolarsTimeZone: std::fmt::Debug + std::fmt::Display {}

#[cfg(feature = "timezones")]
pub trait PolarsTimeZone: chrono::TimeZone + std::fmt::Debug + std::fmt::Display {}

#[cfg(feature = "timezones")]
impl PolarsTimeZone for chrono::FixedOffset {}
#[cfg(feature = "timezones")]
impl PolarsTimeZone for Tz {}
```
and this passes the tests, but then `make pre-commit` fails with
```

```Rust
error[E0277]: the trait bound `chrono::FixedOffset: polars_arrow::PolarsTimeZone` is not satisfied
  --> polars/polars-time/src/date_range.rs:60:60
   |
60 |             date_range_vec(start, stop, every, closed, tu, NO_TIMEZONE)?,
   |             --------------                                 ^^^^^^^^^^^ the trait `polars_arrow::PolarsTimeZone` is not implemented for `chrono::FixedOffset`
   |             |
   |             required by a bound introduced by this call
```

Reply:
for now I've reverted to defining `PolarsTimeZone` in `polars-time/src/lib.rs` and just adding `Debug` and `Display` in two places in `polars-arrow`

if I figure out how to define it in `polars-arrow` I'll add another commit

Reply:
Could you make it work until you are stuck? Then I can try to make a PR on top of this PR. I think it should work.